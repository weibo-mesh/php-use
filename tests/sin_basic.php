<?php


echo "*** Test by calling method or function with its expected arguments ***\n"

$number = 



var_dump(sin( $number ) );


?>
===DONE===
