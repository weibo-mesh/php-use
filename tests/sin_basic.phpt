--TEST--
Test function sin() by calling it with its expected arguments
--SKIPIF--
<?php
?>
--INI--

--FILE--
<?php


echo "*** Test by calling method or function with its expected arguments ***\n"

$number = 



var_dump(sin( $number ) );


?>
===DONE===
--CLEAN--
<?php
?>
--EXPECTF--
===DONE===
