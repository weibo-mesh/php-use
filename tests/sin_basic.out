Parse error: syntax error, unexpected '$number' (T_VARIABLE), expecting ',' or ';' in /Users/idevz/code/z/git/weibo-mesh/php-use/tests/sin_basic.php on line 6